index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+  <header>
+    <nav class="container">
+      <img id="logo" src="img/image_geometry_1.png" alt="">
+      <div id="bars">
+        <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>
+      </div>
+    </nav>
+  </header>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
+  <section class="intro">
+    <h1 id="intro-heading">
+      Hello! I am a software developer!
+    </h1>
+    <p id="para">
+      I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
```

Comment:
- [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
 - [ ] The previous code reviewer asked you to use the `h1` tag, but in a not correct place. The "Hello! I am a software developer!" is part of the paragraph and it was correct when you first submitted the PR. The main title (e.g., "Kashif Saleem") should be an `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly move your name inside the `h1` and move the "Hello..." to inside the `p` tag.

Reply:
**Is this correct now?**
![image](https://user-images.githubusercontent.com/111677605/198273874-fca8eedb-56d9-4b52-a006-ac8ecf5d0ffe.png)
 **For the first suggestion are you implying that I change the tag of (KASHIF SALEEM) from header to section?**